specification/paths/Collections-collection_id.json

```diff
@@ -47,6 +47,15 @@
                           "description": "Indicates whether the collection should be registered at the carrier."
                         }
                       }
+                    },
+                    "relationships": {
+                      "type": "object",
+                      "additionalProperties": false,
+                      "properties": {
+                        "shipments": {
+                          "$ref": "#/definitions/ShipmentRelationship"
```

Comment:
I think this will make the shipment relationship a single resource instead of an array of resources.
Correct me if I'm wrong, but we might have to create a ShipmentsRelationship (where data is an array of shipment identifiers instead of a single object).

Reply:
We do the same for manifests and consolidated shipments. https://carrier-specification.myparcel.com/#tag/Manifests it seems to work there

Reply:
Ah okay. Didn't realize that. Perhaps the relationship should be renamed to ShipmentsRelationship in that case.